implicit declaration of function AfpErr2name

**Describe the bug**

Builds fails due to implicit function declaration

**To Reproduce**

build netatalk in fedora environment

**Expected behavior**

builds successfully

**Environment**

- Server OS: Fedora 43
- Client OS: Fedora 43
- Netatalk Version: 4.4.4.1

**Configuration**
N/A

**Logs**

```
In file included from ../etc/afpd/directory.c:30:
../etc/afpd/directory.c: In function \u2018dirlookup_internal\u2019:
../etc/afpd/directory.c:756:25: error: implicit declaration of function \u2018AfpErr2name\u2019 [-Wimplicit-function-declaration]
  756 |             ntohl(did), AfpErr2name(afp_errno));
      |                         ^~~~~~~~~~~
../include/atalk/logger.h:151:103: note: in definition of macro \u2018LOG\u2019
  151 |             make_log_entry((log_level), (type), __FILE__, type_configs[type].timestamp_us, __LINE__,  __VA_ARGS__); \
      |                                                                                                       ^~~~~~~~~~~
[168/482] gcc -Ietc/afpd/libafpd.a.p -Ietc/afpd -I../etc/afpd -I. -I.. -Iinclude -I../include -Isys -I../sys -Isubprojects/bstring-1.0.3 -I../subprojects/bstring-1.0.3 -Isubprojects/bstring-1.0.3/bstring -I../subprojects/bstring-1.0.3/bstring -I/usr/include/iniparser -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/tinysparql-3.0 -I/usr/include/gio-unix-2.0 -I/usr/lib64/pkgconfig/../../include/dbus-1.0 -I/usr/lib64/pkgconfig/../../lib64/dbus-1.0/include -I/usr/include/json-glib-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/libxml2 -I/usr/include/tirpc -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=c11 -DHAVE_CONFIG_H '-D_U_=__attribute__((unused))' -Wno-pedantic -Wno-extra -Wno-all -Wno-deprecated-declarations -D_GNU_SOURCE -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fPIC -pthread -D_REENTRANT -DWITH_GZFILEOP -DAPPLCNAME '-D_PATH_CONFDIR="/etc/netatalk/"' '-DVERSION="4.4.1"' '-DSERVERTEXT="/var/lib/netatalk/msg/"' '-D_PATH_STATEDIR="/var/lib/netatalk/"' '-D_PATH_AFPDUAMPATH="/usr/lib64/netatalk/"' -MD -MQ etc/afpd/libafpd.a.p/extattrs.c.o -MF etc/afpd/libafpd.a.p/extattrs.c.o.d -o etc/afpd/libafpd.a.p/extattrs.c.o -c ../etc/afpd/extattrs.c
FAILED: [code=1] etc/afpd/libafpd.a.p/extattrs.c.o 
gcc -Ietc/afpd/libafpd.a.p -Ietc/afpd -I../etc/afpd -I. -I.. -Iinclude -I../include -Isys -I../sys -Isubprojects/bstring-1.0.3 -I../subprojects/bstring-1.0.3 -Isubprojects/bstring-1.0.3/bstring -I../subprojects/bstring-1.0.3/bstring -I/usr/include/iniparser -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/tinysparql-3.0 -I/usr/include/gio-unix-2.0 -I/usr/lib64/pkgconfig/../../include/dbus-1.0 -I/usr/lib64/pkgconfig/../../lib64/dbus-1.0/include -I/usr/include/json-glib-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/libxml2 -I/usr/include/tirpc -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -std=c11 -DHAVE_CONFIG_H '-D_U_=__attribute__((unused))' -Wno-pedantic -Wno-extra -Wno-all -Wno-deprecated-declarations -D_GNU_SOURCE -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fPIC -pthread -D_REENTRANT -DWITH_GZFILEOP -DAPPLCNAME '-D_PATH_CONFDIR="/etc/netatalk/"' '-DVERSION="4.4.1"' '-DSERVERTEXT="/var/lib/netatalk/msg/"' '-D_PATH_STATEDIR="/var/lib/netatalk/"' '-D_PATH_AFPDUAMPATH="/usr/lib64/netatalk/"' -MD -MQ etc/afpd/libafpd.a.p/extattrs.c.o -MF etc/afpd/libafpd.a.p/extattrs.c.o.d -o etc/afpd/libafpd.a.p/extattrs.c.o -c ../etc/afpd/extattrs.c
In file included from ../etc/afpd/extattrs.c:28:
../etc/afpd/extattrs.c: In function \u2018afp_listextattr\u2019:
../etc/afpd/extattrs.c:107:13: error: implicit declaration of function \u2018AfpErr2name\u2019 [-Wimplicit-function-declaration]
  107 |             AfpErr2name(afp_errno));
      |             ^~~~~~~~~~~
        ^~~~~~~~~~~
```

**Additional context**

This trivial patch fixes the issue:
https://github.com/knight-of-ni/Netatalk/commit/68f0c1dd3a40dd42182ce4fc8530302b93d085b1

--- a/etc/afpd/directory.c
+++ b/etc/afpd/directory.c
@@ -23,6 +23,7 @@
 
 #include <atalk/adouble.h>
 #include <atalk/afp.h>
+#include <atalk/afp_util.h>
 #include <atalk/cnid.h>
 #include <atalk/errchk.h>
 #include <atalk/fce_api.h>

--- a/etc/afpd/extattrs.c
+++ b/etc/afpd/extattrs.c
@@ -23,6 +23,7 @@
 
 #include <atalk/adouble.h>
 #include <atalk/afp.h>
+#include <atalk/afp_util.h>
 #include <atalk/ea.h>
 #include <atalk/globals.h>
 #include <atalk/logger.h>

